<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentItemWidget</class>
 <widget class="QWidget" name="ComponentItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QToolButton" name="buttonDownloadNew">
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/edit-download.svg</normaloff>:/images/edit-download.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkCanDownload">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelComponentName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Component Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="buttonRemoveFromFile">
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/edit-delete.svg</normaloff>:/images/edit-delete.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="labelComponentUrl">
     <property name="text">
      <string>Component URL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
